ims/exception/file_system_exceptions.py

```diff
@@ -41,6 +41,20 @@ def __str__(self):
         return self.name + " is Busy (Could have Clones)"
 
 
+# this exception should be raised when to remove a protected snapshot
+class SnapshotBusyException(ImageBusyException):
```

Comment:
SnapshotBusyException inherits ImageBusyException and output error message which is more specific